hal-sim/hal_impl/functions.py

```diff
@@ -1023,8 +1035,7 @@ def getMatchTime(status):
         return (hooks.getFPGATime() - hal_data['time']['match_start'])/1000000.0
 
 def waitForDSData():
-    with hooks.ds_cond:
-        hooks.ds_cond.wait()
+    waitForDSDataTimeout(0)
```

Comment:
I think this should be `None`

Reply:
the variable should be?

Reply:
0 should be None, as None is an infinite timeout... I don't know if 0 is.